packages/strapi-helper-plugin/lib/src/components/LabelIconWrapper/index.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+const LabelIconWrapper = styled.span`
```

Comment:
`LabelIconWrapper` sounds a bit weird to me. What do you think of "LabelledIcon" where you could pass an icon and a child that would act as the Icon label?

Reply:
I can rename the component however, I am not sure it's a good idea right now to create a component that will manage the icon, a lot of users have made modifications to the CM and I don't want to introduce to much breaking change before the set up of the design system.

What do you think?

Reply:
I'm not sure to understand why it would be a breaking change? This is a new component nope? 🤔

Reply:
Yes, but if I create a new Label it won't be

Reply:
I don't understand. But anyway, not critical